core/build.gradle.kts

```diff
@@ -48,6 +46,7 @@ dependencies {
     implementation(Libraries.slf4jAPI)
     implementation(Libraries.logbackClassic)
     implementation(Libraries.influxDbClient)
+    compile(Libraries.koin)
```

Comment:
replace compile with implementation

Reply:
this is intentional to propagate koin to other modules since injection takes place in CLI and gradle-plugin

Reply:
use api instead of compile